tests/project/_common/test_movement.py

```diff
@@ -0,0 +1,539 @@
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            include_destructive_artifacts=["incorrect_artifact_name"],
+            overwrite_existing=True,
+        )
+    # Expected ValueError but got RuntimeError instead
+    with pytest.raises(RuntimeError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            exclude_artifacts=["incorrect_artifact_name"],
+            overwrite_existing=True,
+        )
+    # trying to include an additive artifact that is not supported
+    # This is RuntimeError, not ValueError
+    with pytest.raises(RuntimeError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            include_additive_artifacts=[CategorizationArtifacts.CATEGORIZATION_MODEL],
+            overwrite_existing=True,
+        )
+
+
+@mock_api()
+def test_export():
+    client = utils.client.create(**CONFIG["toolbox_test_instance"])
+
+    project = client.projects.by_resource_id(CONFIG["projects"]["minimal_categorization"])
+    path_export_dir = "/home/ubuntu/tamr/projectExports"
+    exclude_list = [
+        CategorizationArtifacts.CATEGORIZATION_VERIFIED_LABELS,
+        CategorizationArtifacts.CATEGORIZATION_FEEDBACK,
+        CategorizationArtifacts.CATEGORIZATION_TAXONOMIES,
+    ]
+
+    op = categorization.export_artifacts(
+        project=project,
+        artifact_directory_path=path_export_dir,
+        exclude_artifacts=exclude_list,
+        asynchronous=False,
+    )
+
+    assert op.succeeded()
+
+
+@pytest.mark.parametrize(
+    "project_to_export, new_project_name, "
+    "include_additive_artifacts, include_destructive_artifacts",
+    [
+        (CONFIG["projects"]["minimal_categorization"], "new_categorization_1", None, None,),
+        (CONFIG["projects"]["minimal_mastering"], "new_mastering_1", None, None,),
+        (CONFIG["projects"]["minimal_golden_records"], "new_golden_records_1", None, None,),
+        (CONFIG["projects"]["minimal_schema_mapping"], "new_schema_mapping_1", None, None,),
+        (
+            CONFIG["projects"]["schema_mapping_with_spaces_in_name"],
+            "new_sm_with_spaces",
+            None,
+            None,
+        ),
+        (
+            CONFIG["projects"]["minimal_categorization"],
+            "new_categorization_2",
+            [CategorizationArtifacts.CATEGORIZATION_VERIFIED_LABELS],
+            [
+                CategorizationArtifacts.CATEGORIZATION_MODEL,
+                CategorizationArtifacts.CATEGORIZATION_TAXONOMIES,
+            ],
+        ),
+        (
+            CONFIG["projects"]["minimal_mastering"],
```

Comment:
You can make the test parameters as passed here simpler to shorten the test filenames.  For instance, instead of
```python
        (
            CONFIG["projects"]["minimal_mastering"],
            "new_mastering_2",
            [MasteringArtifacts.RECORD_PAIR_VERIFIED_LABELS],
            [MasteringArtifacts.CLUSTERING_MODEL, MasteringArtifacts.PUBLISHED_CLUSTERS],
        )  
```
you could have
```python
        (
            "minimal_mastering",
            [MasteringArtifacts.RECORD_PAIR_VERIFIED_LABELS],
            [MasteringArtifacts.CLUSTERING_MODEL, MasteringArtifacts.PUBLISHED_CLUSTERS],
        )  
```
and then in your test have `CONFIG["projects"][project_to_export]` in place of `project_to_export` as it is now.  You can have a dictionary that maps, e.g. `"minimal_mastering"` to `"new_mastering_2"` and use that to look up the correct `new_project_name` value in each case.

Reply:
You solution worked like a charm:
adding
```
      - name: Support longpaths
        run: git config --system core.longpaths true
```
at the beginning of the
```
  Test-Windows:
    ...
    steps:
```
section (around line 70) of .github/workflows/ci.yml